interp/gta.go

```diff
@@ -209,7 +209,7 @@ func (interp *Interpreter) gta(root *node, rpath, importPath string) ([]*node, e
 					// imports in different source files of the same package. Therefore, we suffix
 					// the key with the basename of the source file.
 					name = filepath.Join(name, baseName)
-					if _, exists := sc.sym[name]; !exists {
+					if _, exists := sc.sym[name]; interp.allowRedecl || !exists {
```

Comment:
see my comment for line 236 below, and do the same

Reply:
👍